cpu/stm32/kconfig/f4/Kconfig.models

```diff
@@ -6,6 +6,14 @@
 #
 
 # CPU models
+config CPU_MODEL_STM32F401CCU6
```

Comment:
There is no need to use that level of details in the cpu model name (`u6`). It just give information on the packing type.
Even ST doesn't give that level of details in [their online product list](https://www.st.com/en/microcontrollers-microprocessors/stm32f401.html#products).

Also have a look at #15129 where I already had to adapt the cpu model for the weact-f411ce.

Reply:
I guess I'll just wait for  #15129  to be merged then any happily drop 108c0e57f50ae906f51fc4fd7a5d3f03cea94a10